src/main/java/uk/gov/companieshouse/exemptions/delta/LoggingConfig.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.exemptions.delta;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+/**
+ * Configuration class for logging.
+ */
+@Configuration
+public class LoggingConfig {
```

Comment:
There is a Bean definition for a logger in Config already, static method and field can be moved into there and bean method can be modified to set it.

Reply:
Addressed in latest commit